projects/uswds-components/src/lib/search/search.component.html

```diff
@@ -0,0 +1,13 @@
+ <section [attr.aria-label]="searchSettings.ariaLabel? searchSettings.ariaLabel : 'search component'">
+   <form class="usa-search" role="search" [ngClass]="getClass()">
+     <label class="usa-sr-only" [attr.for]="searchSettings.id?searchSettings.id : 'search' ">Search</label>
+     <input class="usa-input" [attr.id]="searchSettings.id? searchSettings.id : 'search'" type="search" name="search"
+       [placeholder]="searchSettings.placeholder? searchSettings.placeholder : 'type here'" #searchInput [value]="model"
+       (blur)="focusChange($event)" (input)="onValueChange($event.target.value)" (keydown)="onKeydown($event)" />
+     <button class="usa-button" type="submit" (click)="onSubmit(searchInput.value, $event);">
+       <span [ngClass]="[searchSettings?.size === 'small' ? 'usa-sr-only' : 'usa-search__submit-text']">
+         {{(searchSettings?.buttonText) ? searchSettings.buttonText : 'Search'}}
+       </span>
+     </button>
+   </form>
+ </section>
```

Comment:
Could we unpack searchSettings and instead have the properties -  ariaLabel, id, size, buttonText, and placeholder be the inputs? And then define the defaults in ts file rather than through the ternary operations in html.